test/unit/model/stateMachine/connectorConfig.test.ts

```diff
@@ -8,8 +8,6 @@
  *       Yevhen Kyriukha <yevhen.kyriukha@modusbox.com>                   *
  **************************************************************************/
 
-import 'tsconfig-paths/register';
-
 import { ConnectorConfig } from '@app/lib/model/stateMachine/states';
```

Comment:
ts paths support is removed how is this import statement working?

Reply:
it works coz we have this setting in tsconfig: https://github.com/pm4ml/mojaloop-payment-manager-management-api/pull/32/files#diff-b55cdbef4907b7045f32cc5360d48d262cca5f94062e353089f189f4460039e0R37

Reply:
Ok, that's surprising. Without having that in dependencies, we can still import? Anyway if you are sure, lets resolve this comment.